haskell/test/Test/Ligo/BaseDAO/Management/TransferOwnership.hs

```diff
@@ -23,136 +20,134 @@ import Universum
+  => WithOriginateFn 3 m -> WithStorage -> m ()
 rewritePendingOwner withOriginatedFn initialStorage =
-  withOriginatedFn 3 (\(owner:_) -> initialStorage owner) $
-    \[owner, wallet1, wallet2] baseDao -> do
+  withOriginatedFn (\(owner::<_) -> initialStorage owner) $
+    \(owner ::< wallet1 ::< wallet2 ::< Nil') baseDao -> do
       withSender owner . inBatch $ do
-        call baseDao (Call @"Transfer_ownership")
-          (#newOwner :! wallet1)
-        call baseDao (Call @"Transfer_ownership")
-          (#newOwner :! wallet2)
+        (transfer baseDao $ calling (ep @"Transfer_ownership")
+          (#newOwner :! (toAddress wallet1)))
+        (transfer baseDao $ calling (ep @"Transfer_ownership")
+          (#newOwner :! (toAddress wallet2)))
         pure ()
       pendingOwner <- sPendingOwnerRPC <$> (getStorageRPC baseDao)
-      assert (pendingOwner == wallet2) "Pending owner from earlier call was not re-written"
+      assert (pendingOwner == (toAddress wallet2)) "Pending owner from earlier call was not re-written"
 
 invalidatePendingOwner
   :: MonadCleveland caps m
-  => WithOriginateFn m -> WithStorage -> m ()
+  => WithOriginateFn 2 m -> WithStorage -> m ()
 invalidatePendingOwner withOriginatedFn initialStorage =
-  withOriginatedFn 2 (\(owner:_) -> initialStorage owner) $
-    \[owner, wallet1] baseDao -> do
+  withOriginatedFn (\(owner::<_) -> initialStorage owner) $
+    \(owner ::< wallet1 ::< Nil') baseDao -> do
       withSender owner . inBatch $ do
-          call baseDao (Call @"Transfer_ownership")
-            (#newOwner :! wallet1)
-          call baseDao (Call @"Transfer_ownership")
-            (#newOwner :! owner)
+          (transfer baseDao $ calling (ep @"Transfer_ownership")
+            (#newOwner :! (toAddress wallet1)))
+          (transfer baseDao $ calling (ep @"Transfer_ownership")
+            (#newOwner :! (toAddress owner)))
           pure ()
       administrator <- sAdminRPC <$> (getStorageRPC baseDao)
-      assert (administrator == owner) "Pending owner from earlier call was not re-written"
+      assert (administrator == (toAddress owner)) "Pending owner from earlier call was not re-written"
 
 bypassAcceptForSelf
   :: MonadCleveland caps m
-  => WithOriginateFn m -> WithStorage -> m ()
+  => WithOriginateFn 1 m -> WithStorage -> m ()
 bypassAcceptForSelf withOriginatedFn initialStorage =
-  withOriginatedFn 1 (\(owner:_) -> initialStorage owner) $
-    \[owner] baseDao -> do
+  withOriginatedFn (\(owner::<_) -> initialStorage owner) $
+    \(owner ::< Nil') baseDao -> do
       withSender owner $ do
-        call baseDao (Call @"Transfer_ownership")
-          (#newOwner :! (unTAddress baseDao))
-      currentAdmin <- sAdminRPC <$> getStorage @Storage (unTAddress baseDao)
-      assert (currentAdmin == (unTAddress baseDao)) "Admin address was not set"
+        (transfer baseDao $ calling (ep @"Transfer_ownership")
+          (#newOwner :! (toAddress $ chAddress baseDao)))
```

Comment:
```suggestion
          (#newOwner :! toAddress baseDao))
```
Anyway, it's probably easier to do search and replace on `toAddress $ chAddress`, so I won't mention it below.